iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -241,10 +241,10 @@ def createViewer(self, scriptUrl="js/dist/ixbrlviewer.js"):
                     factData["d"] = d
 
             for d, v in f.context.qnameDims.items():
-                if v.memberQname is None:
+                if v.memberQname is None and v.typedMember is not None:
```

Comment:
Swap the if/else to avoid the redundant condition?
```
if v.memberQname is not None:
  ...
elif v.typedMember is not None:
  ...
```
It would also be nice to add test coverage to test_iXBRLViewer.  Perhaps separately since I'm not sure that code is currently structured to make that very convenient.